setup-git-commit-signing.sh

```diff
@@ -67,6 +67,20 @@ else
   middle="\`bpb\` was already configured with a GPG key"
 fi
 
+
+if tea +charm.sh/gum gum confirm -f "shall we add \`bpb\` to \`your github account\`?"; then
+  tea +charm.sh/gum gum format "first, we need to refresh your github with \`write:gpg_key\` scope."
+  # authenticate with gh on web with https and extra permission to write gpg_key
+  tea +github.com/cli/cli gh auth login -h github.com -p https -s write:gpg_key -w
```

Comment:
what happens if they are already auth’d?

Reply:
They need to refresh the token anyways because most likely they don't have the scope: `write:gpg_key`